services/table-geocoder/spec/factories/database.json

```diff
@@ -1,6 +1,6 @@
 {
   "host"      : "localhost",
-  "port"      : 5432,
+  "port"      : 4302,
```

Comment:
Where is this being used and what error is rising?

Maybe this values shouldn't be used at all, that's why I'm asking